docs/madr/decisions/008-mesh-logging.md

```diff
@@ -196,19 +198,19 @@ spec:
   targetRef:
     kind: MeshService
     name: web-backend
-    tags:
-      kuma.io/zone: us-east
   from:
     - targetRef:
         kind: MeshService
         name: web-frontend
       default:
         backends:
-          - type: tcp
-            conf:
+          - tcp:
```

Comment:
Had to change the format because of the `oneof` `struct` issue that we talked about yesterday.